vicare_exporter/metrics.py

```diff
@@ -87,7 +85,7 @@ def extract_feature_metrics(feature: dict, installation_id: str):
     if component_id is not None:
         labels[label_name] = component_id
 
-    for prop in ["value", "active", "shift", "slope", "day", "status"]:
+    for prop in ["value", "active", "shift", "slope", "day", "status","currentDay","lastSevenDays","currentMonth","hours","temperature"]:
```

Comment:
With my heating, I don't actually see/get those properties - so that's why they aren't there 😉 
My intention of using just `day` was that polling the consumption in the e.g. lastSevenDays and currentMonth is pretty redundant if you're already querying the `currentDay` every ~two minutes.
Accumulating monthly/daily consumption can be done afterwards.